.github/workflows/gardener_issue_comment.yml

```diff
@@ -15,6 +16,25 @@ jobs:
     runs-on: ubuntu-latest
     if: contains(github.event.issue.url, 'issues')
     steps:
+      - name: Generate authentication token
+        id: generate_token
+        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a
+        with:
+          app_id: ${{ secrets.GH_APP_DATADOG_VECTOR_CI_APP_ID }}
+          private_key: ${{ secrets.GH_APP_DATADOG_VECTOR_CI_APP_PRIVATE_KEY }}
+
+      - name: Get PR comment author
+        id: comment
+        uses: tspascoal/get-user-teams-membership@v3
+        with:
+          username: ${{ github.actor }}
+          team: 'Vector'
+          GITHUB_TOKEN: ${{ steps.generate_token.outputs.token }}
+
+      - name: Exit if author is team member
+        if: steps.comment.outputs.isTeamMember == 'true'
+        run: exit 0
```

Comment:
I don't think this actually aborts the workflow, does it? I believe each `run` command is in its own subshell so this would just cause this step to pass but still run later ones.

Reply:
yessshhh you are correct. I fixed in the latest.